icu4j/pom.xml

```diff
@@ -14,6 +14,8 @@
   <name>${proj-title} (${project.artifactId})</name>
   <description>International Component for Unicode for Java (ICU4J) is a mature, widely used Java library
     providing Unicode and Globalization support</description>
+  <url>https://icu.unicode.org/</url>
+  <inceptionYear>2001</inceptionYear>
```

Comment:
This is a weird year. ICU4J was started in ca. 1995, and went open source in 1999. What is the reason for 2001?

Reply:
> This is a weird year. ICU4J was started in ca. 1995, and went open source in 1999. What is the reason for 2001?

Comes from `<icu_root>/icu4j/maven/**/pom.xml`
I'll use whatever you say. 1995, or 1999?

Reply:
I like 1995, if it's all the same. :-)

Reply:
Ok, 1995 makes it cool, so go for that ;-)

Reply:
Done, 1995 is it.